.github/workflows/stale-issue-add-needs-testing.yml

```diff
@@ -0,0 +1,17 @@
+name: "Mark old issues as needs confirmation"
+on:
+  schedule:
+  - cron: "45 1 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: "Hi,\nThis issue has gone 180 days without any activity. This means it's time for a check-in to make sure it is relevant today. If you are still experiencing this issue, you can help the project by responding to confirm the problem and by providing any updated reproduction steps.\nThanks for helping out."
+        days-before-stale: 180
```

Comment:
Hi,\nThis issue has gone 180 days without any activity. This means it's time for a check-in to make sure it is relevant today. 

Rephrasing.
Hi, this issue has gone 180 days without any activity. This means it is time for a check-in to make sure it is still relevant.

Reply:
Thanks! Updated.